backends/qualcomm/utils/utils.py

```diff
@@ -54,11 +57,15 @@ def capture_program(
     module: torch.nn.Module,
     inputs: Tuple[torch.Tensor],
 ) -> exir.ExirExportedProgram:
-    ex_prog = exir.capture(
+    exir_exported_program = exir.capture(
```

Comment:
So we should really be moving to export + to_edge api, like here https://github.com/pytorch/executorch/blob/main/backends/xnnpack/test/tester/tester.py#L146 or maybe i missed something. I dont know how you are doing quantization otherwise.